extension.html

```diff
@@ -0,0 +1,119 @@
+              </dd>
+          </dl>
+          <p>The <code>PointerEventInit</code> dictionary is used by the <code>PointerEvent</code> interface's constructor to provide a mechanism by which to construct untrusted (synthetic) pointer events. It inherits from the <code>MouseEventInit</code> dictionary defined in [[!DOM-LEVEL-3-EVENTS]]. The steps for constructing an event are defined in [[!DOM4]]. The <coalescedEventInits> will be used to create coalesced PointerEvents of the generated untrusted synthetic pointer event. The number of coalesced events will be the same as <code>coalescedEventInits</code> list and all of their attributes will be have the same values as the generated event unstrusted synthetic pointer event except those that have a different value in their coalescedEventInits entry. See the <a href="#examples" title="examples">examples</a> for sample code demonstrating how to fire an untrusted pointer event.</p>
+          </p>
+      </div>
+    </section>
+
+    <section class="informative">
+      <h2>Examples</h2>
+      <p>The following are examples that demonstrate how the APIs in this specification might
+      be used</p>
+<pre id="example_1" class="example" title="Creating an untrusted pointer event with coalesced events from script">
+<code>
+var event = new PointerEvent("pointerover",
+  { bubbles: true,
+    cancelable: true,
+    composed: true,
+    pointerId: 42,
+    pointerType: "pen",
+    clientX: 300, 
+    clientY: 500,
+    coalescedEventInits: [{clientX 505}, {clientX: 305, clientY: 505}]
+  });
+// event.getCoalescedEvents().length is 2
+// event.clientX is 300
+// event.clientY is 500
+// event.getCoalescedEvents()[0].clientX is 300
+// event.getCoalescedEvents()[0].clientY is 505
+// event.getCoalescedEvents()[1].clientX is 305
+// event.getCoalescedEvents()[1].clientY is 505
+</code>
+</pre>
+<pre id="example_2" class="example" title="Drawing using coalesced events">
+<code>window.addEventListener("pointermove", function(event) {
+  var listOfAllEvents = event.getCoalescedEvents();
+  listOfAllEvents.push(event);
+  for (let e of listOfAllEvents) {
+    // Draw point for event e
```

Comment:
nit, maybe make this more concrete with code like `drawPoint(e.pageX, e.pageY)`.